client/src/components/ReviewItem.jsx

```diff
@@ -14,7 +14,6 @@ export default class ReviewItem extends React.Component {
 
   componentDidMount() {
     if (this.props.review.review.length > 280) {
-      console.log('hello');
```

Comment:
console.log('component did mount');

Reply:
I was just checking where the problem was. i will delete it.